src/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import BookList from './components/BookList'; // Import your components
+import BookForm from './components/BookForm';
+import Navigation from './components/Navigation';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Navigation />
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <div>
+              {/* Define your home page content here */}
+              <h2>Home Page</h2>
+              <BookList />
+            </div>
+          )}
+        />
+        <Route
+          path="/categories"
```

Comment:
- [ ] nice job so far. Please kindly create a category component for this path. For now, you can just display `under construction` text on the page. Your category page is showing the add book form instead. Nice work